_pages/en_US/updating/futurerestore-help.md

```diff
@@ -36,7 +36,7 @@ To fix this:
 
 This error means that you have not set your generator on your device to that of the blob. In order to solve this problem:
 
-### Getting Started
+
```

Comment:
I'm a bit confused at this change being made here, I feel this maybe should still keep the same subheading and stuff to maintain a somewhat similar style to other pages on the guide.

Reply:
frcoal